artifacts/meminit.csv

```diff
@@ -0,0 +1,3 @@
+0x00000000,0xDEADBEEF,1
```

Comment:
I believe that the memory should be initialized with a state for every node.

Is M=0, S=1, I=3?